packages/compass-aggregations/src/utils/cancellable-aggregation.ts

```diff
@@ -32,37 +29,19 @@ export async function aggregatePipeline({
   if (signal.aborted) {
     return Promise.reject(signal.reason ?? createCancelError());
   }
```

Comment:
Wouldn't it make sense to handle this in the dataService itself?

Reply:
Yes, sounds good. Thanks.

Reply:
i removed it and kept it in when calling the `cancellableOperation`. i don't see any point having it around for every function.

Reply:
Oh, we already have it there, yeah, that's perfect